JitsiConference.js

```diff
@@ -1601,7 +1605,9 @@ JitsiConference.prototype.onCallEnded = function(
     if (jingleSession === this.jvbJingleSession) {
         this.wasStopped = true;
 
-        Statistics.sendEventToAll(SESSION_TERMINATE);
+        const event = createJingleEvent(ACTION_JINGLE_TERMINATE);
+
+        Statistics.sendEventAndLog(event);
```

Comment:
Previously you wrote:
````javascript
Statistics.sendEventAndLog(createJingleEvent(ACTION_JINGLE_RESTART));`
````

Is there are reason that (pretty much) the same source code now looks differently:
````javascript
const event = createJingleEvent(ACTION_JINGLE_TERMINATE);

Statistics.sendEventAndLog(event);
````

From my perspective as a reader of the source code (and maybe the PR) having one and the same thing written in different ways requires unnecessary effort to comprehend.

Reply:
There's no reason, I've made it consistent by removing the constants.